tools/lisa-build-asset

```diff
@@ -60,22 +70,54 @@ def list_assets(recipe_dir):
 def get_asset_recipe(asset, recipe_dir):
     return os.path.join(recipe_dir, '{}.recipe'.format(asset))
 
-def make(asset, actions, arch, toolchain, recipe_dir, build_dir):
+def get_recipe_name(asset):
+    return asset + ".recipe"
+
+def make_native_build(asset, arch, recipe_dir, build_dir, alpine_source_dir, arch_chroot_dir, env):
+    # Install the chroot
+    subprocess.check_call(['sudo', os.path.join(alpine_source_dir, 'alpine-chroot-install'),
```

Comment:
We cannot call `sudo` from a script like that, it can hang depending on the system-level settings of sudo that are outside of our control. If sudo is required, the user must run the whole script as root. (maybe we can try to drop privileges when we can)

Reply:
>  it can hang depending on the system-level settings of sudo that are outside of our control

I am not sure to understand the case. The issue is that the script will hang asking for sudo privilieges ?
On the other hand, running the script as sudo wipes out the lisa environment / functions sourced, preventing the script to run properly.

Reply:
as discussed offline, let's keep sudo. If environment needs to be preserved, we can use `sudo -E`.